test/index.test.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const Knex      = require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  },
+  useNullAsDefault: true
```

Comment:
`Knex:warning - sqlite does not support inserting default values. Set the useNullAsDefault flag to hide this warning. (see docs http://knexjs.org/#Builder-insert).`